templates/form_base.html

```diff
@@ -0,0 +1,59 @@
+{% load static %}
+{% load analytics %}
+{% load i18n %}
+{% load render_bundle from webpack_loader %}
+
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html lang="{{language}}" class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html lang="{{language}}" class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html lang="{{language}}" class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]><!--> <html lang="{{language}}" class="no-js"> <!--<![endif]-->
+  <head>
+      <meta charset="utf-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <title>{% trans "Learning circles" %}</title>
+      <link rel="icon" href="{% static 'images/favicon.ico' %}">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <meta name="description" content="{{ site.description }}">
+      <meta property="og:title" content="Learning circles" />
+      <meta property="og:type" content="website" />
+      <meta property="og:url" content="https://learningcircles.p2pu.org/" />
+      <link rel="canonical" href="">
+      <link rel="shortcut icon" href="{% static 'images/favicon.ico' %}">
+      {% render_bundle 'p2pu-strap' 'css' 'STYLEBUILD' %}
+
+      {% block css %}{% endblock %}
+      <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+      <script src="{% static 'js/vendor/modernizr-2.6.2.min.js' %}"></script>
+      <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
+      <link href="https://unpkg.com/aos@2.3.1/dist/aos.css" rel="stylesheet">
+      {{ form.media }}
```

Comment:
@s-kennedy could we perhaps merge this with the normal base layout and just put the custom bits conditional if needed?